apps/hash-graph/libs/api/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "graph-api"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+hash-status = { workspace = true }
+graph = { workspace = true, features = ["utoipa"] }
+graph-types = { workspace = true }
+temporal-versioning = { workspace = true }
+authorization = { workspace = true }
+type-system = { workspace = true }
+validation = { workspace = true }
+codec = { workspace = true, optional = true }
+
+async-trait = "0.1.74"
+axum = "0.7.2"
+base64 = "0.21.5"
+bytes = "1.5.0"
+error-stack = { workspace = true }
+futures = { workspace = true }
+http-body-util = "0.1.0"
+hyper = "1.0.1"
+include_dir = "0.7.3"
+mime = "0.3.17"
+opentelemetry = "0.21.0"
+opentelemetry_sdk = { version = "0.21.1", features = ["rt-tokio"] }
+sentry = { version = "0.32.0", features = ["tracing", "tower", "tower-http"], default-features = false }
+serde = { version = "1.0.193", features = ['derive'] }
+serde_json = "1.0.108"
+time = { workspace = true }
+tokio = { workspace = true, features = ["macros"], optional = true }
+tokio-postgres = { version = "0.7.10", default-features = false, optional = true }
+tokio-util = { workspace = true, features = ["io"], optional = true }
+tower = "0.4.13"
+tower-http = { version = "0.5.0", features = ["trace"] }
+tracing = "0.1.40"
+tracing-opentelemetry = "0.22.0"
+utoipa = "4.1.0"
+uuid = "1.6.1"
+
+[features]
+test-server = ["dep:tokio", "dep:tokio-postgres", "dep:tokio-util", "dep:codec"]
```

Comment:
Interesting that these dependencies are only required for the test server 👀

Reply:
Yep I was surprised myself 👀